atomic_reactor/constants.py

```diff
@@ -152,6 +152,10 @@
 KOJI_MAX_RETRIES = 120
 KOJI_RETRY_INTERVAL = 60
 KOJI_OFFLINE_RETRY_INTERVAL = 120
+# max retries for skopeo operations
+SKOPEO_MAX_RETRIES = 4
+# the factor for the exponential backoff series - 5, 10, 20, 40 seconds of waiting
```

Comment:
I'd probably wait for at least 2 minutes

Reply:
Ack, increased retries to 5, which makes the maximum possible time 155 seconds